### PR TITLE
CRI-O: Don't mount bpf path since k8s >= 1.11

### PR DESCRIPTION
**Pull request by aanm**
It seems since cri-o 1.11 the `/sys/fs/bpf` is already mounted into the
container. To avoid Cilium from failing its initialization setup, as it
checks if bpf is correctly mounted, the crio kubernetes descriptors
won't have the volume mounted for `/sys/fs/bpf` as it is automatically
done by runc.

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5623)

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

File: examples/kubernetes/1.11/cilium-crio-ds.yaml [hunk 1]
```diff
@@ -42,8 +42,6 @@ spec:
                 - "NET_ADMIN"
```
Comment (ianvernon) [inline examples/kubernetes/1.11/cilium-crio-ds.yaml:42]: Same here, typo in commit message: cri-o 1.11 --> k8s 1.11

Comment (aanm) [reply examples/kubernetes/1.11/cilium-crio-ds.yaml:42]: it's not a typo

Comment (ianvernon) [reply examples/kubernetes/1.11/cilium-crio-ds.yaml:42]: Resolved in 1.1 backport thread for this change.

**Review by ianvernon — APPROVED** [review]

**Comment by ianvernon** [comment]
Build appeared to get stuck in the provisioning steps:
```
17:04:27     k8s2-1.9: Digest: sha256:ab46208466bb204cb628ca940773f00aceccaba919c03e1b61f56999050226e5
17:04:27     k8s2-1.9: Status: Downloaded newer image for k8s1:5000/cilium/cilium-dev:latest
17:14:39 ===================== TEST FAILED =====================
17:14:39 ===================== TEST FAILED =====================
```

Trying again...

**Comment by ianvernon** [comment]
test-missed-k8s

**Comment by aanm** [comment]
test-me-please

**Comment by ianvernon** [comment]
The K8s 1.9 tests failed due to 
https://github.com/cilium/cilium/issues/5638 , which is unrelated to this change. Thus, am merging.

**Pull request merged**